app/Http/Controllers/API/HXLController.php

```diff
@@ -13,17 +15,22 @@
  * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
  */
 
-class HXLController extends Controller
+class HXLController extends RESTController
 {
 
+	protected function getResource()
+	{
+		return 'hxl';
+	}
+
 	/**
 	 * Retrieve a basic information about the API
 	 *
 	 * GET /api
 	 *
 	 * @return void
 	 */
-	public function index()
+	public function index(Request $request)
```

Comment:
You can just remove this as you add real endpoints

Reply:
ah yeah removed somewhere else :( sorry